### PR TITLE
Merge upstream

### PR DESCRIPTION
**Pull request by msgmaxim**
- merged upstream changes
- most translation changes have been ignored

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -166,6 +166,12 @@ The Loki GUI on Windows is 64 bits only; 32-bit Windows GUI builds are not offic
 
     There is no more need to download some special installer from the Qt website, the standard MSYS2 package for Qt will do in almost all circumstances.
 
+    **Note:** There is a known issue that GUI won't lauch properly when building tag v0.13.0.3 with Qt 5.11.2.
```
Comment (Doy-lee) [inline README.md]: The comment about the tag should be v2.0.0

File: get_libwallet_api.sh [hunk 2]
```diff
@@ -16,8 +15,8 @@ if [ ! -d $LOKI_DIR/src ]; then
     git submodule init loki
 fi
 git submodule update --remote
-git -C $LOKI_DIR fetch
-# git -C $LOKI_DIR checkout v0.12.0.0
+# git -C $LOKI_DIR fetch
+git -C $LOKI_DIR checkout dev
```
Comment (Doy-lee) [inline get_libwallet_api.sh]: Keep this to master. It should be if an end user downloads the repository and tries to build, they build against master- or how it was before, it builds against whichever branch is currently checked out (in which case the typical end-user default case is cloning from master).

File: pages/SharedRingDB.qml [hunk 3]
```diff
@@ -106,18 +116,21 @@ Rectangle {
         LabelSubheader {
             Layout.fillWidth: true
             textFormat: Text.RichText
-            text: "<style type='text/css'>a {text-decoration: none; color: #78BE20; font-size: 14px;}</style>" +
-                  qsTr("Blackballed Outputs") + "<a href='#'>(" + qsTr("Help") + ")</a>" + translationManager.emptyString
+            text: "<style type='text/css'>a {text-decoration: none; color: #FF6C3C; font-size: 14px;}</style>" +
```
Comment (Doy-lee) [inline pages/SharedRingDB.qml]: ff6c3c looks like a monero color? Is this intended? 78BE20 is from our colour pallete.

File: README.md (hunk 1 above)
Comment (msgmaxim) [reply README.md]: updated

File: get_libwallet_api.sh (hunk 2 above)
Comment (msgmaxim) [reply get_libwallet_api.sh]: updated

File: pages/SharedRingDB.qml (hunk 3 above)
Comment (msgmaxim) [reply pages/SharedRingDB.qml]: updated

**Pull request merged**